### PR TITLE
Dumb Gun Balance Patch

### PR DESCRIPTION
**Pull request by Neerti**
### General:
* Adds AP rounds for 7.62 and 5.56 mm rounds.  AP rounds have 50 armor pen, but do less damage than standard rounds.
    * Base armor-pen for rifle rounds reduced to 15, from 20.
* Sets groundwork for penalizing one-handing a weapon that requires two hands, based on the weapon itself (e.g. the penalty is a lot less for one-handing an Adv. Egun vs a LWAP).  No weapons require two-hands at this time, due to a lack of twohand sprites.

### Traitor/Mercs:
* AP rounds can be ordered in the uplink.
* .45 lethal rounds can be ordered in the uplink.
* Shotgun slugs, pellets, and beanbags can be ordered in the uplink.
* Adds Egun, Ion, Carbine, Laser Cannon, Xray, and Combat Shotgun to the uplink.

### X-ray
* Capacity reduced to 12 shots, from 20.
* Firing rate reduced to default rate.
    * This is to change the role of the x-ray gun from a straight upgrade from carbine to anti-armor.

### Laser Cannon
* Capacity increased to 6 shots, from 5.
* Now has bonus accuracy, to promote long-ranged fighting.

### Energy Gun
* Firing rate reduced.

### Laser Carbine
* New firing mode added, which fires very weak lasers, but is much more efficient, meant for 'suppressive' fire, or for spider killing.  The lasers do 15 damage but consume 1/4 the charge of a standard 40 damage laser.

### Taser
* Capacity increased to 10 shots.

### Stun Revolver
* Now fires a slightly stronger electrode.

### CODE REVIEW

**Comment by Yoshax** [comment]
The font or formatting you've used is kinda icky.

Also that laser carbine stuff sounds OP as hell, considering how fast you can fire it.

**Comment by Neerti** [comment]
The carbine has the standard firing rate.

**Comment by Anewbe** [comment]
More capacity for tasers much yes, everything else also much yes.

**Comment by Kelenius** [comment]
>Sets groundwork for penalizing one-handing a weapon that requires two hands

Wasn't this already done? Pretty sure I saw this in the code.

>X-ray

Well what's the point now, might as well remove it and replace with a laser gun. Is it really different from a carbine?

>Taser
>Stun revolver

Security buffs :<

**Comment by Neerti** [comment]
>Wasn't this already done? Pretty sure I saw this in the code.

Currently the penalties are hardcoded to be a reduction of a specific amount of accuracy and adding more dispersion.  I'm changing it so that certain weapons are worse to one-hand than others, what I'm gonna call 'one-and-a-half handed' guns.

>Well what's the point now, might as well remove it and replace with a laser gun. Is it really different 

from a carbine?
Xray's role has been changed from 'best gun ever' to 'good if fighting armored targets'.

>Security buffs :<

There's a sentiment that tasers are useless against anything that's not an unarmed assistant, which I feel is due to the really small capacity of the taser, as the capacity has otherwise not changed from /tg/station, back when tasers shot electrodes which instantly stunned you.  With RNG playing a factor in hitting and needing more than one shot to stun someone, five shots feels very low, and as a result, 'real weapons' are preferred.

**Comment by Kelenius** [comment]
>Xray's role has been changed from 'best gun ever' to 'good if fighting armored targets'.

Let me rephrase that: is it different enough from the carbines to justify having an another gun?

**Comment by Mechoid** [comment]
I can't make comments on the specific lines regarding them, though borg tasers seem to have been overlooked in the adjustment of tasers.

**Comment by Mark9013100** [comment]
Aren't borg tasers a child of regular tasers?

**Comment by Mechoid** [comment]
Yes, but they are coded to have one more shot than normal officer tasers.

**Comment by Neerti** [comment]
So this has been sitting for a month, going to bring this up on the forums for discussion.
Changed the PR description to look nicer, otherwise has not changed since the last version.
